go.mod

```diff
@@ -1,15 +1,17 @@
 module github.com/glifio/glif/v2
 
-go 1.21
+go 1.22
+
+toolchain go1.22.0
 
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.7
 	github.com/briandowns/spinner v1.23.0
 	github.com/ethereum/go-ethereum v1.12.0
 	github.com/fatih/color v1.15.0
-	github.com/filecoin-project/go-address v1.1.0
-	github.com/filecoin-project/go-state-types v0.14.0
-	github.com/filecoin-project/lotus v1.28.1
+	github.com/filecoin-project/go-address v1.2.0
+	github.com/filecoin-project/go-state-types v0.15.0
+	github.com/filecoin-project/lotus v1.30.0
```

Comment:
Updating these deps didn't break anything in here with actor codes when running against the public node did it?

Reply:
I reverted go.mod to the older versions